### DISCUSSION TITLE
How to Update directly rather than reinstall

### DISCUSSION THREAD

**Discussion by hello-zhanghao** [gitpod-io/openvscode-server#223] [Q&A]
The openvscode-server is very suitable for coding anywhere and anytime in the browser,but when I want used the latest version, I need to uninstall the older and reinstall the latest, how can I update the version of openvscoe directly?

**Comment by jeanp413** [comment]
@hello-zhanghao what do you mean by uninstall and reinstall? If you are using the docker images then whenever there's a new release just pull the new image

**Reply by hello-zhanghao** [reply]
Thanks! I install the openvscode in my docker container, because I do not want to reinstall the extension when upgrade the openvscode. so I do not want to pull the new image. Today I reinstall the latest 1.62.0, and I find that the seeting and extension is also exist, so it is very lucky.

**Reply by jeanp413** [reply]
I see, btw starting from version 1.62.0 you can now tell the server to install the extensions on startup e.g:
`server.sh --install-extension my.extension --install-builtin-extension my.other-extension`
